advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx

```diff
@@ -169,7 +173,7 @@ EOF
 
 `$MY_IP` and `$MY_NAME` are specific to the local host. Otherwise, the `patroni.yml` configuration is the same on all Patroni nodes.
 
-Patroni expects to find the `postgres` binary in the `bin_dir` location, while the EDB Postgres Advanced Server binary is called `edb-postgres`. Create a symbolic link to let Patroni find that binary:
+Patroni expects to find the `postgres` binary in the `bin_dir` location, while the EDB Postgres Advanced Server binary is called `edb-postgres`. As the `postgresql.bin_name` setting was introduced in the [`3.0.3` release](https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-303), remove it from the configuration if you are running an older version and create a symbolic link to let Patroni find that binary:
```

Comment:
Specify that it's an older version of Patroni:
"... if you are running an older version of Patroni and create..."